modules/strelka/1.0/config/default.yaml

```diff
@@ -0,0 +1,36 @@
+lcr-modules:
+    
+    strelka:
+        inputs:
+            # Available wildcards: {seq_type} {genome_build} {tumour_id} {normal_id} {pair_status}
```

Comment:
I would move this line to right before `candidate_indel` and add a separate comment here with:
```
# Available wildcards: {seq_type} {genome_build} {sample_id}
```